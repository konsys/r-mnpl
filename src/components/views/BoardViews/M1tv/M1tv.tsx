import React from "react";
export const M1tv = (props: any) => (
  <>
    <div className="table-body-board-m1tv">
      <div className="table-body-board-m1tv-controls">
        <div mnpl-size="small" mnpl-action="rewind_back_60">
          –60
        </div>
        <div mnpl-size="small" mnpl-action="rewind_back_15">
          –15
        </div>
        <div mnpl-size="small" mnpl-action="rewind_back_5">
          –5
        </div>
        <div
          mnpl-size="big"
          mnpl-action="play"
          style={{ display: "none" }}
        ></div>
        <div mnpl-size="big" mnpl-action="pause"></div>
        <div mnpl-size="small" mnpl-action="rewind_fwd_5">
          +5
        </div>
        <div mnpl-size="small" mnpl-action="rewind_fwd_15">
          +15
        </div>
        <div mnpl-size="small" mnpl-action="rewind_fwd_60">
          +60
        </div>
      </div>
      <div className="table-body-board-m1tv-timer">
        <div></div>
        <div></div>
      </div>
      <div className="table-body-board-m1tv-speed">
        <div className="_btn">–</div>
        <div className="_val">M!</div>
        <div className="_btn">+</div>
      </div>
    </div>
  </>
);
