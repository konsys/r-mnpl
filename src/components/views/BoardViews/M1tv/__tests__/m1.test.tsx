import { M1tv } from "../M1tv";
import React from "react";
import { shallow } from "enzyme";

describe("Test m1", () => {
  it("should render properly", () => {
    expect(shallow(<M1tv />)).toMatchSnapshot();
  });
});
