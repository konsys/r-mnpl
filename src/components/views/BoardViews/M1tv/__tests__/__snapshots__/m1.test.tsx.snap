// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test m1 should render properly 1`] = `ShallowWrapper {}`;
